src/components/ContextChart/index.js

```diff
@@ -99,7 +98,7 @@ const ContextChart = ({
         s.yDomain,
         Axes.y(domainsByItemId[s.collectionId || s.id])
       ) ||
-      calculateDomainFromData(s.data, s.yAccessor, s.y0Accessor, s.y1Accessor);
+      [0,0];
```

Comment:
Could you please explain a bit the change here?

Reply:
Poor (lazy) mans ignore ;) i should have fixed it (like fixing types and stuff) or ts-ignored it. Not remove it lol